docs/dev/tools/cli.md

```diff
@@ -1190,6 +1191,24 @@ viam.service.vision.v1.VisionService.GetClassificationsFromCamera
 
 # restart a part of a specified machine
 viam machines part restart --machine=123 --part=456
+
```

Comment:
Is there a reason these examples use a space and not an `=` like the other examples in this code block? Seems they should be consistent with above as well as with the template where you have `=<>` if both work?

Reply:
changed